app/move/app/new/fields/move-type.js

```diff
@@ -5,7 +5,7 @@ const moveType = {
   name: 'move_type',
   fieldset: {
     legend: {
-      text: 'fields::move_type.label',
+      html: `<h2 class="govuk-heading-m govuk-!-margin-bottom-1">Move to</h2>`,
```

Comment:
I think it'd be best if we keep using the translations file for the text, something like this:

```suggestion
      html: `<h2 class="govuk-heading-m govuk-!-margin-bottom-1">${i18n.t('fields::move_type.label')}</h2>`,
```

with `import i18n from '../../../../../config/i18n'` at the top

Reply:
@Thource Sure thing! Cheers Adam 🙏